### ISSUE TITLE
[a11y]: Pagination, PaginationNav - There is no label for the prev and next buttons

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#14683]
[Original report](https://ibm-studios.slack.com/archives/C054WK4ES5S/p1694709381258889)

> For the Previous and Next buttons in the Pagination, there is no label in iPhone Safari
>
> There is no label for the prev and next buttons in VO iPhone Safari
